File: Unidad 3/C#/P24_U3.cs
using System;

namespace P24_U3 // Note: actual namespace depends on the project name.
{
    internal class Program
    {
        static void Main(string[] args)
        {
            int n;

            do
            {

                Console.WriteLine("Ingresa un número:");
                n = Convert.ToInt32(Console.ReadLine());
                if (n % 2 == 0)
                {
                    Console.WriteLine("Número par");
                }
                else
                {
                    Console.WriteLine("Número impar");
                }
            } while (n > 0);
        }
    }
}

File: Unidad 3/JAVA/P24_U3.java
/*
 * Click nbfs://nbhost/SystemFileSystem/Templates/Licenses/license-default.txt to change this license
 */

package com.mycompany.p24_u3;

import java.util.Scanner;

/**
 *
 * @author LaboratorioLTI2
 */
public class P24_U3 {

    public static void main(String[] args) {
        Scanner sc = new Scanner (System.in);
        
        int n;
        
        do{
            
            System.out.println("Ingresa un número:");
        n=sc.nextInt();
            if (n%2==0){
                System.out.println("Número par");
            }
            else {
                System.out.println("Número impar");
            }   
        } while (n>0);
    }
}
